wire-ios/Wire-iOS Tests/ConversationCell/StartedConversationCellTests.swift

```diff
@@ -30,7 +30,7 @@ final class StartedConversationCellTests: ConversationMessageSnapshotTestCase {
         UIColor.setAccentOverride(.vividRed)
         SelfUser.setupMockSelfUser(inTeam: UUID())
 
-        mockSelfUser = SelfUser.current as? MockUserType
+        mockSelfUser = SelfUser.provider?.providedSelfUser as? MockUserType
```

Comment:
Do we really want these changes for the unit tests as well?

Reply:
Yes, because then I could remove the problematic convenient property of `current` and it can not be used in future anymore.

Reply:
I agree, we need those changes in Unit Tests as well.